File: SoloLearn Codes/Easter Eggs/EasterEggs.java
import java.util.Scanner;

public class EasterEggs
{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int total = sc.nextInt();
        int me = sc.nextInt();
        int friend = sc.nextInt();
        System.out.println(total <= me + friend ? "Candy Time" : "Keep Hunting");
    }
}

File: SoloLearn Codes/Easter Eggs/EasterEggs.cs
using System;


namespace Sololearn
{
    class Program
    {
        static void Main(string[] args)
        {
            int total = Convert.ToInt32(Console.ReadLine());
            int me = Convert.ToInt32(Console.ReadLine());
            int friend = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine(total <= me + friend ? "Candy Time" : "Keep Hunting");
        }
    }
}

